### PR TITLE
Add perf action to CI

### PR DESCRIPTION
**Pull request by vorosl**

### CODE REVIEW

File: src/jit/PerfDump.cpp [hunk 1]
```diff
@@ -310,12 +310,12 @@ uint32_t PerfDump::dumpByteCode(InstructionListItem* item)
             fprintf(m_sourceFile, "%c R%d", prefix, static_cast<int>(i - paramCount));
         }
 
-        switch (VARIABLE_TYPE(operand->ref)) {
-        case Operand::Immediate:
+        switch (VARIABLE_TYPE(*operand)) {
+        case Instruction::Immediate:
```
Comment (zherczeg) [inline src/jit/PerfDump.cpp:314]: I think this is ConstPtr

https://github.com/Samsung/walrus/blob/main/src/jit/Compiler.h#L205

Did you tested the patch?

Comment (vorosl) [reply src/jit/PerfDump.cpp:314]: Yes, this failed, I'll fix it

**Review by zherczeg — APPROVED** [review]
LGTM

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**